tests/ViewTest.elm

```diff
@@ -0,0 +1,33 @@
+module ViewTest exposing (suite)
+
+import Test exposing (Test, describe, test)
+import Test.Html.Event as Event
+import Test.Html.Query as Query
+import Test.Html.Selector exposing (containing, tag, text)
+import Main exposing (view, Msg(..))
```

Comment:
As I understand, elm is against `..` imports. (prevent shadowing variables). Even so I think not all messages are view specific (I know, right now they are, but they don't have to - we will add internal messages, messages from ports etc.).

Reply:
I'm gonna change that, you're right!

Reply:
It seems to work a bit differently than we thought.

<img width="909" alt="Screenshot 2020-04-29 at 23 23 22" src="https://user-images.githubusercontent.com/1434406/80648441-790bad00-8a70-11ea-8bb3-161dd75cb968.png">